src/SFA.DAS.AssessorService.Application.Api/Services/OrganisationMergingService.cs

```diff
@@ -0,0 +1,280 @@
+            var mosv = new MergeOrganisationStandardVersion()
+            {
+                StandardUid = sourceOrganisationStandardVersion.StandardUId,
+                Version = sourceOrganisationStandardVersion.Version,
+                Replicates = replicates,
+                OrganisationStandardId = sourceOrganisationStandardVersion.OrganisationStandardId,
+                Status = sourceOrganisationStandardVersion.Status,
+                Comments = sourceOrganisationStandardVersion.Comments,
+                DateVersionApproved = sourceOrganisationStandardVersion.DateVersionApproved,
+                EffectiveFrom = sourceOrganisationStandardVersion.EffectiveFrom,
+                EffectiveTo = sourceOrganisationStandardVersion.EffectiveTo,
+            };
+
+            return mosv;
+        }
+        private MergeOrganisationStandardDeliveryArea CreateMergeOrganisationStandardDeliveryArea(string replicates, OrganisationStandardDeliveryArea sourceOrganisationStandardDeliveryArea)
+        {
+            var mosda = new MergeOrganisationStandardDeliveryArea()
+            {
+                Replicates = replicates,
+                DeliveryAreaId = sourceOrganisationStandardDeliveryArea.DeliveryAreaId,
+                OrganisationStandardId = sourceOrganisationStandardDeliveryArea.OrganisationStandardId,
+                Status = sourceOrganisationStandardDeliveryArea.Status,
+                Comments = sourceOrganisationStandardDeliveryArea.Comments,
+            };
+
+            return mosda;
+        }
+
+
+        private void MergeOrganisationStandardsAndVersions(Organisation primaryOrganisation, Organisation secondaryOrganisation, Guid createdByUserId, DateTime secondaryStandardsEffectiveTo)
+        {
+            // @ToDo: a bit monolithic - can this be refactored in to something more readable?
+
+            // Grab the primary contact for the primary organisation because we're going to need it.
+            var primaryContact = _dbContext.Contacts.FirstOrDefault(c => c.EndPointAssessorOrganisationId == primaryOrganisation.EndPointAssessorOrganisationId && c.Email == primaryOrganisation.PrimaryContact);
+
+            // Read all the standards from the secondary organisation.
+
+            foreach(var secondaryOrganisationStandard in secondaryOrganisation.OrganisationStandards)
+            {
+                // Does the primary organisation have this standard already?
+                var primaryOrganisationStandard = primaryOrganisation.OrganisationStandards.FirstOrDefault(pos => pos.StandardCode == secondaryOrganisationStandard.StandardCode);
+
+                if (null == primaryOrganisationStandard)
+                {
+                    // No - so add the standard to the primary organisation.
+
+                    primaryOrganisationStandard = new OrganisationStandard()
+                    {
+                        DateStandardApprovedOnRegister = secondaryOrganisationStandard.DateStandardApprovedOnRegister,
```

Comment:
Should this date be the date the secondary was approved, or "now" that's when the primary will be enabled from the merge date?